client/src/js/components/bhGridLoadingIndicator.js

```diff
@@ -4,6 +4,7 @@ angular.module('bhima.components')
     loadingState : '<',
     emptyState   : '<',
     errorState   : '<',
+    messageState : '<?',
```

Comment:
It looks like this message only applies to the `emptyState`.  If so, a nicer binding would be `emptyStateMessage`.  That way, we could also have an `errorStateMessage` and `loadingStateMessage` if we need them.  The bindings would look something like this:
```html
<bh-grid-loading-indicator

  loading-state="$ctrl.loading"
  loading-state-message="I'm thinking about it ... "

  error-state="$ctrl.error"
  error-state-message="Something is terribly wrong."

  empty-state="$ctrl.rows.length === 0"
  empty-state-message="I have nothing to show you, my friends">
</bh-grid-loading-indicator>
```

Reply:
You could also then make it a `@` binding and remove the requirement for a listener.